elastic/kibana
# Handle cluster_block_exception during reindexing the TM index

Resolves: https://github.com/elastic/response-ops-team/issues/249

This PR increases task claiming interval in case of `cluster_block_exception` to avoid generating too many error during TM index reindexing.

## To verify:

- Run your local Kibana,
- Create a user with `kibana_system` and `kibana_admin` roles
- Logout and login with your new user
- Use below request to put a write block on TM index.
   `PUT /.kibana_task_manager_9.0.0_001/_block/write`
- Observe the error messages and their occurring interval on your terminal.
- Use below request on the Kibana console to halt write block.
```
PUT /.kibana_task_manager_9.0.0_001/_settings
{
  "index": {
    "blocks.write": false
  }
}
```
